### ISSUE TITLE
Curriculum Overview, All Pages - the Drop-Down Menu on the Right Not Visible

### ISSUE THREAD

**Issue by kwahlenmayer** [Simon-Initiative/oli-torus#4417]
**Describe the bug**
Author report: As seen in the below screenshot, when you click the right dropdown arrow, the popup menu with "Options", "Move to", etc is hidden and you must use the toolbar to scroll over and down to access it. New users may not realize it because it is hidden. Also, it is a little annoying.

Admin notes: This image appears to be from a view such as Curriculum Overview, All Pages. I cannot replicate the horizontal scroll bar being so wide that the drop-down menu is to the right of my view as shown in the image in either Chrome or Firefox Windows which is what the user uses. I did ask for more details which I will add if hear more information: Is this still happening for you? Is it from authoring, all pages? Chrome or Firefox Windows? Does it improve if you zoom out (CTRL and -)?

![image](https://github.com/Simon-Initiative/oli-torus/assets/24357828/dcfe2a1e-2ae5-44dd-b694-bda0ddafc40e)

**Comment by kwahlenmayer** [comment]
Update, additional information from author:
So I recreated that search. It is still partially blocked but more visible. Is there a way to just eliminate the scroll bar on the bottom so the entire screen is visible no matter the browser size? It did help when I zoomed out so more of the screen was visible. I'm using Google Chrome.

![image](https://github.com/Simon-Initiative/oli-torus/assets/24357828/d1405bd8-8531-4f3f-9e45-1d184a2948cf)
